modules/ROOT/pages/monitoring-dashboards.adoc

```diff
@@ -284,6 +288,8 @@ From this dashboard, you can click a <<Access the Server Monitoring Dashboard,se
 
 == See Also
 
+* xref:monitoring::index.adoc[Anypoint Monitoring Overview]
+* https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/cloudhub-api/1.0.11/console/method/%231708/[CloudHub API - dashboardStats]
```

Comment:
Consider reworking topic titles so that they do not use punctuation (CloudHub API - dashboardStats).